vignettes/adrs.Rmd

```diff
@@ -360,46 +503,30 @@ here and below, in our examples we show these as `ADRS` parameters, but
 they could equally be achieved via `ADSL` dates or `ADEVENT` parameters.
 If you prefer to store as an ADSL date, then the function
 `admiral::derive_var_extreme_dt()` could be used to find the date of
-first `PD` as a variable, rather than as a new parameter record. All the
-parameter derivation functions that use these dates are flexible to
-allow sourcing these from any input source using
-`admiral::date_source()`. See examples below.
+first `PD` as a variable, rather than as a new parameter record.
 
 ## Derive Response Parameter {#rsp}
 
-The next required step is to define the source location for this newly
-derived `PD` date.
+The function `adiral::derive_extreme_event()` can then be used to find the date
```

Comment:
```suggestion
The function `admiral::derive_extreme_event()` can then be used to find the date
```

Reply:
this typo exists in few places - please fix all

Reply:
All fixed